### DISCUSSION TITLE
Naming of the filter_var/FilterState "unremovable" arguments

### DISCUSSION THREAD

**Discussion by gogonzo** [insightsengineering/teal.slice#283] [Polls]
We need to decide on the proper naming of the following properties:

- fixed (already implemented) - not interactive filter (user can't change what is selected)
- unremovable - filter which can't be removed nor disabled.

@chlebowa @lcd2yyz @asbates @donyunardi @kartikeyakirar @MarcinKosinski

**Comment by lcd2yyz** [comment]
While choose the options, please also consider what should be the default behavior for the filters, and make sure both argument design are consistent in terms of default values (eg. both should be FALSE or both should be TRUE). For example, we shouldn't pick `selectable` and `locked` as a combination.

**Comment by gogonzo** [comment]
`locked` it is 🥇
